datasets/ONS-Trade-in-goods/exports.py

```diff
@@ -1,24 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
 #!/usr/bin/env python
 # coding: utf-8
 # %% [markdown]
 # # Country by commodity exports
 
+
 # %%
+
+# In[2]:
+
+
+import pandas as pd
+import numpy as np
 from gssutils import *
 import json
+from gssutils.metadata import THEME
+from gssutils.metadata import *
+from databaker.framework import *
+
+
+# In[3]:
+
+
+import datetime
+
+cubes = Cubes("info.json")
+title = "Trade in goods: country-by-commodity exports"
+
+
+# In[4]:
+
+
+with open ('info.json') as file:
+    info = json.load(file)
+
+
+# In[5]:
+
+
+landingPage = info['landingPage'][0]
+landingPage
+
+
+# In[6]:
 
-landingPage = json.load(open('info.json'))['landingPage']
-display(landingPage)
 
 scraper = Scraper(landingPage)
+scraper.dataset.family = info['families']
```

Comment:
this is correct and makes perfect sense ...but ... only because there's 2 urls and we don't have proper handling for that with the `Scraper(seed="info.json")` argument. 

So nicely done :smile: but be concious of why we're having to pass a specific url to Scraper and try and use `Scraper(seed="info.json')` in all other scenarios.